Added Readme for helm (#383)
kubernetes-sigs/scheduler-plugins: manifests/install/charts/as-a-second-scheduler/README.md

# Scheduler-plugins as a second scheduler in cluster

## Installation

Quick start instructions for the setup and configuration of as-a-second-scheduler using Helm.

### Prerequisites

- [Helm](https://helm.sh/docs/intro/quickstart/#install-helm)

### Installing the chart

#### Install chart using Helm v3.0+

```bash
$ git clone git@github.com:kubernetes-sigs/scheduler-plugins.git
$ cd scheduler-plugins/manifests/install/charts
$ helm install scheduler-plugins as-a-second-scheduler/
```

> 

Reviewer: ````suggestion
Verify that scheduler and plugin-controller pod are running properly.

```bash
$ kubectl get deploy -n scheduler-plugins
NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
scheduler-plugins-controller   1/1     1            1           7s
scheduler-plugins-scheduler    1/1     1            1           7s
```
````

| --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
| `scheduler.name`                        | Scheduler name                                                                                                                            | `scheduler-plugins-scheduler`                           |
| `scheduler.image`                       | Scheduler image                                                                                                                           | `k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.22.6`   |
| `scheduler.namespace`                   | Scheduler namespace                                                                                                                       | `scheduler-plugins`                                     |

> | `scheduler.replicaCount`                | Scheduler replicaCount                                                                                                                    | `1`                                                     |

Reviewer 2: ```suggestion
| `scheduler.replicaCount`                | scheduler-plugins replicas                                                                                                                | `1`                                                     |
```

| `controller.name`                       | Controller name                                                                                                                           | `scheduler-plugins-controller`                          |
| `controller.image`                      | Controller image                                                                                                                          | `k8s.gcr.io/scheduler-plugins/controller:v0.22.6`       |
| `controller.namespace`                  | Controller namespace                                                                                                                      | `scheduler-plugins`                                     |    
| `controller.replicaCount`               | Controller replicaCount                                                                                                                   | `1`                                                     |

> | `plugins.enabled`                       | Plugins enabled                                                                                                                           | `["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]` |

Reviewer 2: ```suggestion
| `plugins.enabled`                       | All Plugins are enabled by default. Plugins enabled                                                                                                                           | `["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]` |
```

> | `global.queueSort`                      | global queueSort, needs to be globally enabled once                                                                                       | `["Coscheduling"]`                                      |

Reviewer 2: Please remove `global` from all parameters' descriptions. 
```suggestion
| `global.queueSort`                      |  THe default `queueSort` Plugin, needs to be enabled once                                                                                       | `["Coscheduling"]`                                      |
```

> | `global.extensions.preFilter`           | global extensions preFilter                                                                                                               | `["Coscheduling", "CapacityScheduling"]`                |

Reviewer 2: The description should be something like:
 ```markdown
 `preFilter` extension config. This is be used if the `preFilter` plugin is enabled.
 ```

Please apply the same for the rest of all extensions. parameters
